### DISCUSSION TITLE
Mounting a Child Device to a Parent Device Bay

### DISCUSSION THREAD

**Discussion by adrian-enspired** [netbox-community/netbox#8694] [Q&A] [answered]
I know this is a very basic question, but I'm not finding an answer in the docs, here in discussions, searching google, or by just clicking around the UI. If I'm overlooking something obvious, my apologies - but please point it out  : )

- I've created the parent device type, with its device bays, created an instance of that device, and mounted it in the rack.
- I've created the child device type and an instance of it. 
- I cannot find the way to mount it in one of the parent device instance's bays - i.e., [step # 5 in this answer](https://github.com/netbox-community/netbox/issues/216#issuecomment-231081853).

**Comment by candlerb** [comment, marked as answer]
* Navigate to the parent device instance
* Click on the "Device Bays" tab
* Next to an empty bay, click the green "+" button (hover and it says "Install Device")

**Comment by adrian-enspired** [comment]
Thanks

I don't see a "Device Bays" tab on the Device instance page - only "Device," "Config Context," "Journal," and "Change Log."

The Device Type record does have a "Device Bays" tab with all the bays I created, and is defined as a "parent" device type.

**Comment by candlerb** [comment]
Maybe you added the device bays on the device type *after* you created the device instance?  Components are copied from the device type to the device instance at the time the instance is created only; subsequent changes to the device type have no effect on existing instances.

**Comment by adrian-enspired** [comment]
Interesting to know that templates are literally _copied_ and not just referenced.

I looked at the change log tabs of the device and the template, and i did create the device _after_ i finished adding the device bays, so don't know what's going on there. but I created a new instance and it did copy the bays, so... problem solved.  

thanks!
